bug in the sanity check after maxfilters. (#336)
mne-tools/mne-bids-pipeline: docs/source/changes.md

- The summary report didn't use the cleaned epochs for showing the effects of
  ICA.
  ({{ gh(320) }} by {{ authors.hoechenberger }})

> - The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered data

Reviewer: ```suggestion
- The sanity check comparing the rank of the experimental data and the rank of the empty-room after Maxwell-filtering did not use the maxfiltered data
```

> - The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered matrices

Reviewer 2: ```suggestion
- The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered data
```

>   (by {{ authors.agramfort }}, {{ authors.hoechenberger }} and {{ crsegerie }}).

Reviewer: ```suggestion
  (by {{ authors.agramfort }}, {{ authors.hoechenberger }}, and {{ crsegerie }}).
```
